question/classes/bank/modifier_name_column.php

```diff
@@ -20,6 +20,8 @@
  * @package   core_question
  * @copyright 2009 Tim Hunt
  * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ * @deprecated since Moodle 4.0
```

Comment:
deprecation info will go to the class document

Reply:
Moved depreciation in class documentation